purchase/app/purchase/CheckoutAddressFormData.java

```diff
@@ -0,0 +1,233 @@
+package purchase;
+
+import io.sphere.sdk.models.Base;
+import play.data.validation.Constraints;
+
+public class CheckoutAddressFormData extends Base {
+    public interface Validation {}
+
+    private String csrfToken;
```

Comment:
I didn't add any line of code, just moved things around. I left the clean up phase for future PR so I wouldn't pollute this one.